cfn-resources/go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/tidwall/pretty v1.2.1
 	go.mongodb.org/atlas v0.36.0
 	go.mongodb.org/atlas-sdk/v20231115002 v20231115002.1.0
```

Comment:
Can we remove this SDK version?
```suggestion
```

Reply:
Since the new SDK has breaking changes, for now we need to keep both versions